packages/chart/src/components/BarChart/components/BarChart.Vertical.tsx

```diff
@@ -244,6 +244,8 @@ export const BarChartVertical = () => {
                   )
                   // End Confidence Interval Variables
 
+                  const paddingForBarLabel = 10
```

Comment:
can you change this to, makes it a little more obvious that it's a constant.

const BAR_LABEL_PADDING = 10